src/event/event_dispatcher.cpp

```
@@ -33,19 +34,19 @@ void EventDispatcher::addEventListener(EventListener* l,
   }
 
   auto vec = listeners[eventType];
-  auto it = vec.begin();
+  auto it  = vec.begin();
   if ((it = find(vec.begin(), vec.end(), l)) == vec.end())
```

Clang (14.0.6)

[error] an assignment within an 'if' condition is bug-prone

(bugprone-assignment-in-if-condition,-warnings-as-errors)